### DISCUSSION TITLE
Docs: Increase visibility of modules

### DISCUSSION THREAD

**Discussion by kolibril13** [ManimCommunity/manim#2791] [Suggestions and Proposals]
Now that we have a good visibility of the classes, I think modules should also be easier to find.
Therefore, two ideas:
As for the module-page visibility from classes: We should add two new navigation links to class pages, linking super-class and module.
E.g.
https://docs.manim.community/en/latest/reference/manim.mobject.matrix.Matrix.html
should link to 
https://docs.manim.community/en/latest/reference/manim.mobject.matrix.html

As we have these nice diagrams from @AntonBallmaier 
https://docs.manim.community/en/latest/reference.html#mobject-inheritance-graph
maybe it is also possible to add the modules there:
![image](https://user-images.githubusercontent.com/44469195/119222069-d26ad200-baf2-11eb-9679-7cf5c75b0b9f.png)

**Comment by behackl** [comment]
If we continue restructuring the modules, I'd eventually be in favour of removing the separate HTML pages for classes, and rendering class documentation on the same page as the module they are contained in.

(For inspiration how this could look like, here is an example for documentation of a SageMath module: https://doc.sagemath.org/html/en/reference/asymptotic/sage/rings/asymptotic/growth_group.html -- keep in mind that navigation in our documentation theme would be even easier, as we have a ToC of the page in the sidebar on the right.)
